databroker_pack/_utils.py

```diff
@@ -106,6 +113,10 @@ class PipeToCat:
     def __init__(self, host, path):
         self._host = host
         self._path = path
+        # Ensure directory exists.
+        subprocess.run(
+            ["ssh", host, "mkdir", "-p", str(Path(path).parent)], capture_output=True
+        )
```

Comment:
It's a new feature I missed during my leave. What's the purpose? Remote databroker-packing? Will paramiko be more native here?